### PR TITLE
chore: use github actions

### PR DESCRIPTION
**Pull request by llorllale**
for #83

Signed-off-by: George Aristy <george.aristy@gmail.com>

### CODE REVIEW

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
This pull request [#174](https://github.com/llorllale/cactoos-matchers/issues/174) is assigned to @andreoss[/z](https://www.0crat.com/u/andreoss), here is [why](https://www.0crat.com/footprint/C63314D6Z/908f52d4-3fbd-4c5b-8416-a0763e17f657); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @victornoel[/z](https://www.0crat.com/u/victornoel) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @andreoss[/z](https://www.0crat.com/u/andreoss)[/z](https://www.0crat.com/u/andreoss) didn't make CR comments

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
```
Comment (andreoss) [inline .github/workflows/build.yml:26]: @llorllale I wonder if this notice is necessary. CI configurations for rultor and travis usually lack any copyright information.

**Comment by andreoss** [comment]
@llorllale @victornoel Looks good, but it was merged prior i was assigned (Still got my points)

**Pull request merged**